packages/cli/src/commands/mock.ts

```diff
@@ -20,6 +20,18 @@ const mockCommand: CommandModule = {
           boolean: true,
           default: false,
         },
+        ['callback-delay']: {
+          description: 'Number of seconds to wait before executing callbacks',
+          default: '0',
+          string: true,
+          coerce: coerceRange,
+        },
+        ['callback-count']: {
+          description: 'How many times a callback should be executed',
+          default: '1',
+          string: true,
```

Comment:
The first example out of my head is a subscription to status updates about an invoice. We have such an example in our docs: https://github.com/stoplightio/prism/blob/master/examples/callbacks/payment-service.oas3.yaml#L68

So in above example, after subscribing to status updates about an invoice we can receive possibly three notifications - `created`, `paid`, `failure`.

Reply:
Ok I see what you mean — protect against duplicates and out of order notifications, right?

Reply:
I am not sure about count myself. @StefanDywersant I totally see what you're getting at but I wonder if it would be better to just fire off the callback multiple times if there are multiple named examples, with each one respecting the delay? Or, we give them a chance to trigger specific named examples, something? 

The implementation is not clear for me yet, maybe we could just make this be about the delay and see what our users ask for? There's other things we can do :)

Reply:
@XVincentX that's exactly one of the goals; if folks want, let's put some unexpectedness to their implementation

> but I wonder if it would be better to just fire off the callback multiple times if there are multiple named examples

@philsturgeon it sounds very reasonable from user perspective however, it is super hard to implement. Let's image a `notification` model with multiple enum fields and prism doing `n`x`m`x`...` combinations out of it... one can get some new gray hair from that ;)

I think the current approach has a good balance between simplicity of use (if someone needs that) and flexibility.

Reply:
> The implementation is not clear for me yet, maybe we could just make this be about the delay and see what our users ask for? There's other things we can do :)

Yeah I tend to agree that we haven't really sit and discussed the implementation; on this point, I would rather implement it later instead of providing basic functionality now and then change it with a breaking change. Maybe we could refrain from this now?